examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala

```diff
@@ -51,7 +52,7 @@ object CarbonExample {
         .config("spark.sql.warehouse.dir", warehouse)
         .config("javax.jdo.option.ConnectionURL",
           s"jdbc:derby:;databaseName=$metastoredb;create=true")
-        .getOrCreate()
+        .getOrCreateCarbonSession()
```

Comment:
Do not modify this and change file name to SparkSessionExample. And change CarbonExampleNew to CarbonSessionExample.